src/codem/preprocessing/preprocess.py

```diff
@@ -231,6 +231,8 @@ def _normalize(self) -> None:
             raise RuntimeError(
                 "self.transform is not initialized, you run the prep() method?"
             )
+        # this step gets tripped up by GCS since the transform numbers are going to be tiny!
+        breakpoint()
         scale = np.sqrt(self.transform[0] ** 2 + self.transform[1] ** 2)
```

Comment:
If we're concerned with precision/accuracy of this calculation, we can replace this with:

```python
scale = math.hypot(self.transform[0:2])
```

In theory, this call should be more accurate when we're dealing with very small values.